cleo/ui/exception_trace.py

```diff
@@ -51,9 +51,7 @@ class Highlighter:
     }
 
     KEYWORDS = set(keyword.kwlist)
-    BUILTINS = set(
```

Comment:
it looks like `__builtins__` can be different types based on https://docs.python.org/3/reference/executionmodel.html#builtins-and-restricted-execution

>By default, when in the [__main__](https://docs.python.org/3/library/__main__.html#module-__main__) module, __builtins__ is the built-in module [builtins](https://docs.python.org/3/library/builtins.html#module-builtins); when in any other module, __builtins__ is an alias for the dictionary of the [builtins](https://docs.python.org/3/library/builtins.html#module-builtins) module itself.

Reply:
Changed it to `builtins` module, as suggested by docs.

Reply:
💯